src/gui/parametersdialog.cpp

```diff
@@ -837,13 +839,13 @@ QString ParametersDialog::getErrorLevelNodeText(const ErrorInfo &errorInfo) cons
     switch (errorInfo.exitCode()) {
         case ExitCodeSystemError: {
             if (errorInfo.exitCause() == ExitCauseFileAccessError) {
-                return tr("Can't access item %1.<br>"
-                          "Please fix the write permissions and restart the synchronization.")
-                    .arg(errorInfo.path());
+                return tr(
+                    "Can't access item.<br>"
+                    "Please fix the write permissions and restart the synchronization.");
             }
 
             if (errorInfo.exitCause() == ExitCauseMoveToTrashFailed) {
-                return tr("Move to trash failed for item %1").arg(errorInfo.path());
+                return tr("Move to trash failed.");
```

Comment:
We should decide on the use of the word  "trash" or "bin".

Reply:
I agree with you. Upon checking, I found three places where we use "trash" and three others where we use "bin". Since this pull request is for version 3.6.3 which will be released today, I have created a Jira issue to discuss this matter for version 3.6.4.

Reply:
KDESKTOP-1105